### ISSUE TITLE
Add support for gstreamer 1.0

### ISSUE THREAD

**Issue by jplatte** [squentin/gmusicbrowser#76]
I think this shouldn't have top priority, but it would be nice if one didn't have to install gstreamer 0.10 to use gmb.

**Comment by squentin** [comment]
fe20c1a4c79fa2be
done :)

**Comment by jplatte** [comment]
Awesome! I want to test it, but I don't know which package contains the gstreamer1 perl bindings. `perl-gstreamer` on Arch Linux is dependent on `gstreamer0.10`. After installing the git version and removing the gstreamer ugly and ugly-plugins packages I can't play .mp3's anymore so it seems gstreamer1 is not detected. How do I find the correct packages to install?

**Comment by squentin** [comment]
You just need perl-glib-object-introspection and the introspection data for gstreamer, not sure which package have those in archlinux, it might even be included with the gstreamer libs. On ubuntu it's gir1.2-gstreamer-1.0 it contains the file /usr/lib/girepository-1.0/Gst-1.0.typelib

You'll probably need some gstreamer plugins too, at least gst-plugins-base, which also has some introspection data (gir1.2-gst-plugins-base-1.0 in ubuntu) that gmb optionally use /usr/lib/girepository-1.0/GstPbutils-1.0.typelib (for launching installation of missing codecs) and /usr/lib/girepository-1.0/GstVideo-1.0.typelib (for visuals)

These is a perl gstreamer package (https://metacpan.org/release/GStreamer1) but I chose to not use it for now, as it is currently basically only 1 instruction using Glib::Object::Introspection to load 4 librairies, 3 of them I don't use, and I would still need to load the 2 mentioned above (GstPbutils and GstVideo).

**Comment by jplatte** [comment]
Thanks for the insight! I found the right package just a moment before your comment and edited mine.
I'll leave a comment on the gmusicbrowser-git AUR package to add the introspection package as optdepend :)

Thanks for this update, it's really appreciated! I still have to leave most gstreamer0.10 stuff installed because it's a dependency of a dependency of a dependency of Pidgin, but at least I got rid of the -ugly and -ugly-plugins packges! :D
